src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/INuGetProjectManagerService.cs

```diff
@@ -71,8 +71,10 @@ public interface INuGetProjectManagerService : IDisposable
             bool includePrelease,
             DependencyBehavior dependencyBehavior,
             IReadOnlyList<string> packageSourceNames,
-            VersionRange versionRange,
-            CancellationToken cancellationToken);
+            VersionRange? versionRange,
+            CancellationToken cancellationToken,
+            string? newMappingID = null,
+            string? newMappingSource = null);
```

Comment:
CA1068 usually asks for CancellationToken to be the last parameter of a method. I'm not sure why it's disabled in our repo.